lib/bolt.rb

```diff
@@ -2,4 +2,5 @@
 
 module Bolt
   require 'bolt/executor'
+  require 'bolt/orch_executor'
```

Comment:
This is just for messing with run_task and isolating it a bit. I expect this will live in the executor

Reply:
Is work to refactor the executor ticketed somewhere?